### DISCUSSION TITLE
firefox vertical tabs

### DISCUSSION THREAD

**Discussion by thedolphinisdead** [soulhotel/FF-ULTIMA#185] [Q&A] [answered]
For some reason, in the sidebar, the vertical tabs from firefox seem to "overrule" the one from ultima, meaning it looks weird and i don't get any of the fancy buttons.
![image](https://github.com/user-attachments/assets/d83b2026-1b9e-4e71-a094-d4331f9c811a)

**Comment by soulhotel** [comment, marked as answer]
create an issue

![image](https://github.com/user-attachments/assets/6320328a-9827-46a6-b5de-cac0df2a4e1b)
